src/navigation/tabs/tab/PTab.vue

```diff
@@ -3,7 +3,10 @@
         <ul class="tab-item-wrapper" :class="{stretch}">
             <li v-for="(tab, idx) in tabItems" :key="tab.name"
                 :class="{active: activeTab === tab.name}"
-                @click="handleClickTab(tab, idx)"
+                role="tab"
+                tabindex="0"
```

Comment:
```suggestion
                :tabindex="0"
```
For `number` type, How about this?